conf/simulator/moab_simparam.yaml

```diff
@@ -23,4 +22,5 @@ simulator:
       "config_initial_pitch",
       "config_initial_roll"
     ]
-  policy: bonsai
+  policy: random
+  logging: enable
```

Comment:
you've added `logging` here but not in the other `simulator/*.yaml` files. Still checking the rest of the files but we should keep the examples consistent.